components/prize_pool/wikis/trackmania/prize_pool_custom.lua

```diff
@@ -43,6 +47,18 @@ function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
 		Variables.varDefault('tournament_type')
 	)
 
+	local worldTourPoints = Array.filter(placement.parent.prizes, function (prize)
+		if prize.type == PRIZE_TYPE_POINTS and prize.data.title == PRIZE_TITLE_WORLD_TOUR then
+			return true
+		end
+	end)[1]
+
+	if worldTourPoints then
+		CustomPrizePool.addWTDatapoint(lpdbData, placement:getPrizeRewardForOpponent(opponent, worldTourPoints.id))
+	end
+
+	Variables.varDefine(lpdbData.participant:lower() .. '_prizepoints', lpdbData.extradata.prizepoints)
+
```

Comment:
So it would be something like replacing this by
```lua
lpdbData.extradata.pointsTitle = prize.data.title
```
only ?

Reply:
For pages with teamcards, these will need to set it as well

Reply:
So I need to add the line
```lua
lpdbData.extradata.pointsTitle = prize.data.title
```
both in PrizePool and in TeamCard ? And remove the code I commented ?